include/tvision/internal/linuxcon.h

```diff
@@ -3,7 +3,7 @@
 
 #include <internal/platform.h>
 
-#ifdef __linux__
+#if defined(__linux__) || defined(__FreeBSD__)
```

Comment:
Is this because the "linux console" also works on FreeBSD, or for any other reason?

Reply:
Yes, adding the `defined(__FreeBSD__)` seems to allow the "linux console" code to work. The library and demos compile, and the demos do seem to be functional

Reply:
The linux console strategy does not work on FreeBSD (`source/platform/linuxcon.cpp` does not compile). But there was a bug in this header that could be worked around by your change. This strategy was never chosen, bare Ncurses is used instead (with lack of mouse, Shift detection, etc.).